### DISCUSSION TITLE
Can Harfbuzz be used in a micro-controller with FreeRTOS as OS ?

### DISCUSSION THREAD

**Discussion by emoreac-microej** [harfbuzz/harfbuzz#3195] [Q&A] [answered]
Hello Harfbuzz developers,

I would like to know, 
is it possible to compile Harfbuzz as a static library for the target Cortex-M33 micro-controller with FreeRTOS as RTOS?

Indeed, I would like to confirm with you that if Harfbuzz has necessarily the need of an OS like Windows, MacOS, Linux or not.
Then, It would be interesting to know if this library can be compiled using the IAR compiler instead of GCC.

Thanks in advance for your help.

Best regards,
Erwan

**Comment by behdad** [comment, marked as answer]
We don't know but we don't know of any reason why it shouldn't be possible. You need a compiler that supports C++11. That's the only requirement.
